HW_1/task_02.py

```diff
@@ -0,0 +1,13 @@
+# Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+nums = int(input('Введите трехзначное число: '))
+sum = 0
+for i in range (nums, -1, -1):
```

Comment:
У вас очень много ненужных итераций будет, а нужно всего 3. Тут лучше использовать цикл `while` или так:
```suggestion
for _ in str(nums):
```